test/vic-v2/00-all-fields.e2e.spec.js

```diff
@@ -0,0 +1,68 @@
+const E2eHelpers = require('../e2e/e2e-helpers');
+const Timeouts = require('../e2e/timeouts');
+const PageHelpers = require('../e2e/vic-helpers');
+const testData = require('./schema/maximal-test.json');
+
+const runTest = E2eHelpers.createE2eTest(
+  (client) => {
```

Comment:
What's `client`?

Reply:
It's the interface for Nightwatch, the browser testing library we use: http://nightwatchjs.org/